src/Model/ChangeStreamIterator.php

```diff
@@ -109,7 +101,7 @@ final public function commandStarted(CommandStartedEvent $event): void
         }
 
         $this->batchPosition = 0;
-        $this->batchSize = null;
+        $this->batchSize = 0;
```

Comment:
Noted that this would have left `batchSize` null only if the command failed since `commandSucceeded()` sets it; however, there's no real need to do that since it violates the typing. Also, it has no functional impact on how this is used by `isAtEndOfBatch()`.